### ISSUE TITLE
Existing recovery account recovery on iPhone Xr

### ISSUE THREAD

**Issue by jacogr** [paritytech/parity-signer#261]
Ok, so latest toy, latest Signer.

Steps -

1. Empty accounts screen went to "recover account"

2. Spent the next 5 mins typing in my 34-word Parity-style recovery phrase

3. While entering this, around half-way the words started being behind the keyboard, so I couldn't actually see what I was typing. (Hence spending 5mins, I'm a slow phone typer and needed to check that auto-correct didn't do anything stupid with each word)

4. Once done, I couldn't move out of the box - the keyboard covers half my phrase and the remainder of the screen. No way to close the keyboard to press the button.

5. So... I just went out.

6. Then decided, maybe I will try again.

7. Clicked recover.

8. The recovery screen opened... and well, there is a bug (but it helped me recover from the first issue) - the old phrase I spent 5 mins typing in was still visible. Great! No keyboard, so could press next and add the PIN.

TL;DR Seems to be 2 issues, but the second one helped me recover from the first.

**Comment by jacogr** [comment]
Actually don't even need to type in the whole phrase, just starting is enough to let the keyboard get in the way and overlap the box.

![image](https://user-images.githubusercontent.com/1424473/61526032-e8506d00-aa19-11e9-8118-10b44bf51fd7.png)

So from there, just have no way to minimize the keyboard to move on (Tried "Return")
